src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -125,33 +125,79 @@
         [:div {:style {:clear "both"}}]])}))
 
 
+(defn- modal-background [state]
```

Comment:
I like that you are specifying visibility. It's not enforced in ClojureScript, but I think it adds useful documentation.